src/Sql/Sql/help/Restore-AzSqlInstanceDatabase.md

```diff
@@ -94,6 +94,13 @@ Restore-AzSqlInstanceDatabase [-FromGeoBackup] [-ResourceGroupName] <String> [-I
  [<CommonParameters>]
 ```
 
+### LongTermRetentionBackupRestoreParameter
+```
+Restore-AzSqlInstanceDatabase [-FromLongTermRetentionBackup] [-SubscriptionId <String>] [-ResourceId] <String>
+ -TargetInstanceDatabaseName <String> -TargetInstanceName <String> -TargetResourceGroupName <String> [-AsJob]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
 ## DESCRIPTION
 The **Restore-AzSqlInstanceDatabase** cmdlet restores an instance database from a geo-redundant backup or a point in time in a live database.
```

Comment:
The **Restore-AzSqlInstanceDatabase** cmdlet restores an instance database from a geo-redundant backup or a long term retention backup or a point in time in a live database.

Reply:
oops